### PR TITLE
Change how time zone is passed, Fixed issue #29

### PR DESCRIPTION
**Pull request by IngridGdesigns**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require('dotenv').config();
 const fetch = require('node-fetch');
-const Moment = require('moment');
+const Moment = require('moment-timezone');
```
Comment (kevindigo) [inline index.js:3]: Should there be a package.json change that goes with this?

File: README.md [hunk 2]
```diff
@@ -47,10 +47,26 @@ Log in to GitHub and [register a new personal access token](https://github.com/s
...
+For example: 
+``` .env
+TIMEZONE=“Etc/GMT+6”
+```
+
+would output:
+```Users that contributed between Tue Mar 31 2020 00:00:00 GMT-0600 and Tue Apr 07 2020 23:59:59 GMT-0600 
+```
+
+For further reading visit [moment-timezone](https://momentjs.com/timezone/docs) and [List of tz](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
```
Comment (kevindigo) [inline README.md:62]: We should probably have a blank line or two here (whatever matches the rest of the file).

File: README.md [hunk 3]
```diff
@@ -47,10 +47,26 @@ Log in to GitHub and [register a new personal access token](https://github.com/s
...
 - The CSV you input will be turned into an array, so the numbers for the CSV columns are zero-indexed. If you choose not to use an alternate id, you can put the same column number in both CSV_COLUMN_NUMBER_FOR_GITHUB_ID and CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID.
 
+
+By default, the time window uses UTC-0 (same as GMT). If that's acceptable, leave TIMEZONE_OFFSET as an empty string. If you want your time to be local, provide a UTC offset here.
```
Comment (kevindigo) [inline README.md:66]: This section no longer applies, so can be removed, right?

File: .env.template [hunk 4]
```diff
@@ -1,5 +1,5 @@
 GITHUB_TOKEN = ""
-TIMEZONE_OFFSET = ""
+TIMEZONE ="America/Puerto_Rico"
```
Comment (kevindigo) [inline .env.template:2]: This is an interesting default! :P 

We should probably default to a major city, to maximize the chances that relatively few users will have to change it. I would suggest Los Angeles, to pick up the west coast, but could see an argument for New York or Chicago, or perhaps Tokyo. 

Or perhaps even better would be to default to Etc/UTC, as it is neutral, and would be a good choice for a global organization.

File: README.md (hunk 3 above)
Comment (IngridGdesigns) [reply README.md:66]: nooo, I forgot about that, will remove now

File: .env.template (hunk 4 above)
Comment (IngridGdesigns) [reply .env.template:2]: got it!

Comment (IngridGdesigns) [reply .env.template]: I added the changes, and noticed that If i don't add the commits specifically in VSCode, (this time I used the terminal) it wont add you as a collaborator.

File: index.js (hunk 1 above)
Comment (IngridGdesigns) [reply index.js:3]: done

**Pull request merged**